src/bot/messaging.rs

```diff
@@ -61,6 +71,71 @@ impl Bot {
         self.post_message(&path, &params)
     }
 
+    /// API call whick will send the requested chat action to your bot.
+    pub fn send_chat_action(&self, chat_id: &i64, action: &ChatAction) -> Result<bool> {
+        let action = get_chat_action(action);
+        println!("{:?}", &action);
+        let path = ["sendChatAction"];
+        let url = ::construct_api_url(&self.bot_url, &path);
+        let params = [("chat_id", chat_id.to_string()),
+            ("action", action)];
+        let mut data = self.client.post(&url).form(&params).send()?;
+        println!("{:?}", &data);
+
+        let rjson: Value = check_for_error(data.json()?)?;
+        let message_json = rjson.get("result").ok_or(JsonNotFound)?;
+        println!("{:?}", message_json);
+        if message_json.as_bool().unwrap() {
+            Ok(true)
+        } else {
+            Ok(false)
```

Comment:
Why is it returning `false`? The docs say it returns `true` on success, you should try to return the contents from the JSON response.